cypress/e2e/tests/pages/manager/pod-security-admissions.spec.ts

```diff
@@ -0,0 +1,108 @@
+import PodSecurityAdmissionsPagePo from '@/cypress/e2e/po/pages/cluster-manager/pod-security-admissions.po';
+import PromptRemove from '@/cypress/e2e/po/prompts/promptRemove.po';
+import * as path from 'path';
+import * as jsyaml from 'js-yaml';
+
+describe('Pod Security Admissions', { testIsolation: 'off', tags: ['@manager', '@adminUser'] }, () => {
+  const podSecurityAdmissionsPage = new PodSecurityAdmissionsPagePo('_');
+  const downloadsFolder = Cypress.config('downloadsFolder');
+  const runTimestamp = +new Date();
+  const policyAdmissionName = `e2e-pod-security-admission-name-${ runTimestamp }`;
+  const policyAdmissionDescription = `e2e-pod-security-admission-description-${ runTimestamp }`;
+
+  before(() => {
+    cy.login();
+  });
+
+  beforeEach(() => {
+    cy.viewport(1380, 720);
+  });
+
+  it('can create a policy security admission', () => {
+    PodSecurityAdmissionsPagePo.navTo();
+    podSecurityAdmissionsPage.waitForPage();
+    podSecurityAdmissionsPage.create();
+    podSecurityAdmissionsPage.createPodSecurityAdmissionForm().waitForPage();
+    podSecurityAdmissionsPage.createPodSecurityAdmissionForm().name().set(policyAdmissionName);
+    podSecurityAdmissionsPage.createPodSecurityAdmissionForm().description().set(policyAdmissionDescription);
```

Comment:
would we be able to set some admission control and exemptions fields and confirm they get sent correctly? same goes for edit

Reply:
this is in progress

Reply:
Oh I forgot to mention that this is done. See changes here: https://github.com/rancher/dashboard/pull/10161/commits/110beb8aff8bcf600dffa9071241005a1d0e1569